File: z_cs_py_yapilanlar/CKomisyon.cs
public class CKomisyon
{
    CTrader Trader = null;

    ~CKomisyon()
    {

    }

    public CKomisyon()
    {

    }

    public CKomisyon Initialize(dynamic Sistem, CTrader Trader)
    {
        this.Trader = Trader;

        return this;
    }

    public CKomisyon Reset(dynamic Sistem)
    {
        return this;
    }

    public int Hesapla(dynamic Sistem, int BarIndex)
    {
        int result = 0;

        int i = BarIndex;

        Trader.Status.KomisyonFiyat = Trader.Lists.KomisyonIslemSayisiList[i] * Convert.ToSingle(Trader.Status.KomisyonCarpan) * Trader.Status.KomisyonVarlikAdedSayisi;
        Trader.Lists.KomisyonFiyatList[i] = Trader.Status.KomisyonFiyat;

        return result;
    }
}

File: z_cs_py_yapilanlar/CKomisyon.py
class CKomisyon:
    def __init__(self):
        self.Trader = None

    def __del__(self):
        pass

    def initialize(self, Sistem, Trader):
        self.Trader = Trader
        return self

    def reset(self, Sistem):
        return self

    def hesapla(self, Sistem, BarIndex):
        result = 0
        i = BarIndex
        self.Trader.Status.KomisyonFiyat = self.Trader.Lists.KomisyonIslemSayisiList[i] * float(self.Trader.Status.KomisyonCarpan) * self.Trader.Status.KomisyonVarlikAdedSayisi
        self.Trader.Lists.KomisyonFiyatList[i] = self.Trader.Status.KomisyonFiyat
        return result
